src/apps/chifra/internal/blocks/handle_count.go

```diff
@@ -0,0 +1,102 @@
+// Copyright 2021 The TrueBlocks Authors. All rights reserved.
+// Use of this source code is governed by a license that can
+// be found in the LICENSE file.
+
+package blocksPkg
+
+import (
+	"context"
+
+	"github.com/TrueBlocks/trueblocks-core/src/apps/chifra/pkg/output"
+	"github.com/TrueBlocks/trueblocks-core/src/apps/chifra/pkg/rpcClient"
+	"github.com/TrueBlocks/trueblocks-core/src/apps/chifra/pkg/types"
+)
+
+func (opts *BlocksOptions) HandleCounts() error {
+	// Don't do this in the loop
+	meta, err := rpcClient.GetMetaData(opts.Globals.Chain, opts.Globals.TestMode)
+	if err != nil {
+		return err
+	}
+	if opts.Globals.TestMode {
+		meta.Latest = 2000100
+	}
+
+	ctx, cancel := context.WithCancel(context.Background())
+
+	// Note: Make sure to add an entry to enabledForCmd in src/apps/chifra/pkg/output/helpers.go
+	fetchData := func(modelChan chan types.Modeler[types.RawBlockCount], errorChan chan error) {
+		for _, br := range opts.BlockIds {
+			blockNums, err := br.ResolveBlocks(opts.Globals.Chain)
+			if err != nil {
+				errorChan <- err
+				cancel()
+				return
+			}
+
+			for _, bn := range blockNums {
+				finalized := meta.Age(bn) > 28
```

Comment:
Yes. I do. In fact, I think we maybe want to remove it as it's actually confusing and completely arbitrary. I documented it, but I'm not sure what side effects it might have.

https://github.com/TrueBlocks/trueblocks-core/issues/2667